kong/templates/nginx_kong.lua

```diff
@@ -145,12 +139,6 @@ server {
         proxy_set_header      Host               $upstream_host;
         proxy_set_header      Upgrade            $upstream_upgrade;
         proxy_set_header      Connection         $upstream_connection;
-        proxy_set_header      X-Forwarded-For    $upstream_x_forwarded_for;
-        proxy_set_header      X-Forwarded-Proto  $upstream_x_forwarded_proto;
-        proxy_set_header      X-Forwarded-Host   $upstream_x_forwarded_host;
-        proxy_set_header      X-Forwarded-Port   $upstream_x_forwarded_port;
-        proxy_set_header      X-Forwarded-Path   $upstream_x_forwarded_path;
-        proxy_set_header      X-Forwarded-Prefix $upstream_x_forwarded_prefix;
         proxy_set_header      X-Real-IP          $remote_addr;
```

Comment:
As this is related to `X-Forwarded-For` let's have same treatment to it too (X-Real-IP). It can be set similarly where we do `set_header` for `X-Forwarded-*`